tests/cogs/test_new_user_join.py

```diff
@@ -0,0 +1,71 @@
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+
+from otter_welcome_buddy.cogs import new_user_joins
+
+
+@pytest.fixture
+def mock_bot():
+    mock_bot = AsyncMock()
+    return mock_bot
+
+
+@pytest.fixture
+def mock_msg_fmt():
+    mock_msg_fmt = MagicMock()
+    mock_msg_fmt.bot_is_ready = MagicMock()
+    return mock_msg_fmt
+
+
+@pytest.fixture
+def mock_debug_fmt():
+    mock_debug_fmt = MagicMock()
+    mock_debug_fmt.welcome_message = MagicMock()
+    return mock_debug_fmt
+
+
+@pytest.mark.asyncio
+async def test_cogSetup_registerCommand(mock_bot, mock_msg_fmt, mock_debug_fmt):
```

Comment:
qq: Why do u need mock_msg_fmt and mock_debug_fmt here? The AsyncMock doesn't patch all the function?